<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="5" total="12" passed="2" failed="5" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-04-29T17:39:46Z" name="Suite1" finished-at="2021-04-29T17:40:02Z" duration-ms="16409">
    <groups>
    </groups>
    <test started-at="2021-04-29T17:39:46Z" name="TestNgSmoke" finished-at="2021-04-29T17:40:02Z" duration-ms="16409">
      <class name="com.automation.test.LoginTest">
        <test-method is-config="true" signature="beforeSuite()[pri:0, instance:com.automation.test.LoginTest@6ab778a]" started-at="2021-04-29T17:39:46Z" name="beforeSuite" finished-at="2021-04-29T17:39:46Z" duration-ms="13" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method is-config="true" signature="beforemethod()[pri:0, instance:com.automation.test.LoginTest@6ab778a]" started-at="2021-04-29T17:39:46Z" name="beforemethod" finished-at="2021-04-29T17:39:48Z" duration-ms="2074" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforemethod -->
        <test-method signature="LoginFailure()[pri:0, instance:com.automation.test.LoginTest@6ab778a]" started-at="2021-04-29T17:39:48Z" name="LoginFailure" finished-at="2021-04-29T17:39:49Z" duration-ms="1339" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginFailure -->
        <test-method is-config="true" signature="logout(org.testng.ITestResult)[pri:0, instance:com.automation.test.LoginTest@6ab778a]" started-at="2021-04-29T17:39:49Z" name="logout" finished-at="2021-04-29T17:39:50Z" duration-ms="693" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=LoginFailure status=SUCCESS method=LoginTest.LoginFailure()[pri:0, instance:com.automation.test.LoginTest@6ab778a] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method is-config="true" signature="beforemethod()[pri:0, instance:com.automation.test.LoginTest@6ab778a]" started-at="2021-04-29T17:39:50Z" name="beforemethod" finished-at="2021-04-29T17:39:51Z" duration-ms="1105" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforemethod -->
        <test-method signature="loginSuccess()[pri:0, instance:com.automation.test.LoginTest@6ab778a]" started-at="2021-04-29T17:39:51Z" name="loginSuccess" finished-at="2021-04-29T17:39:52Z" duration-ms="1227" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginSuccess -->
        <test-method is-config="true" signature="logout(org.testng.ITestResult)[pri:0, instance:com.automation.test.LoginTest@6ab778a]" started-at="2021-04-29T17:39:52Z" name="logout" finished-at="2021-04-29T17:39:53Z" duration-ms="727" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginSuccess status=SUCCESS method=LoginTest.loginSuccess()[pri:0, instance:com.automation.test.LoginTest@6ab778a] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
      </class> <!-- com.automation.test.LoginTest -->
      <class name="com.automation.test.DragandDrop">
        <test-method is-config="true" signature="beforemethod()[pri:0, instance:com.automation.test.DragandDrop@1dac5ef]" started-at="2021-04-29T17:39:57Z" name="beforemethod" finished-at="2021-04-29T17:39:58Z" duration-ms="1071" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforemethod -->
        <test-method signature="DragDrop()[pri:0, instance:com.automation.test.DragandDrop@1dac5ef]" started-at="2021-04-29T17:39:58Z" name="DragDrop" finished-at="2021-04-29T17:39:58Z" duration-ms="1" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.automation.test.DragandDrop.DragDrop(DragandDrop.java:15)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DragDrop -->
        <test-method is-config="true" signature="logout(org.testng.ITestResult)[pri:0, instance:com.automation.test.DragandDrop@1dac5ef]" started-at="2021-04-29T17:39:58Z" name="logout" finished-at="2021-04-29T17:39:59Z" duration-ms="703" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=DragDrop status=FAILURE method=DragandDrop.DragDrop()[pri:0, instance:com.automation.test.DragandDrop@1dac5ef] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
      </class> <!-- com.automation.test.DragandDrop -->
      <class name="com.automation.test.ContextMenu">
        <test-method is-config="true" signature="beforemethod()[pri:0, instance:com.automation.test.ContextMenu@77f80c04]" started-at="2021-04-29T17:39:55Z" name="beforemethod" finished-at="2021-04-29T17:39:56Z" duration-ms="1173" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforemethod -->
        <test-method signature="Checkboxes()[pri:0, instance:com.automation.test.ContextMenu@77f80c04]" started-at="2021-04-29T17:39:56Z" name="Checkboxes" finished-at="2021-04-29T17:39:56Z" duration-ms="4" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.automation.test.ContextMenu.Checkboxes(ContextMenu.java:15)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Checkboxes -->
        <test-method is-config="true" signature="logout(org.testng.ITestResult)[pri:0, instance:com.automation.test.ContextMenu@77f80c04]" started-at="2021-04-29T17:39:56Z" name="logout" finished-at="2021-04-29T17:39:57Z" duration-ms="704" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Checkboxes status=FAILURE method=ContextMenu.Checkboxes()[pri:0, instance:com.automation.test.ContextMenu@77f80c04] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
      </class> <!-- com.automation.test.ContextMenu -->
      <class name="com.automation.test.DropDown">
        <test-method is-config="true" signature="beforemethod()[pri:0, instance:com.automation.test.DropDown@5c90e579]" started-at="2021-04-29T17:39:59Z" name="beforemethod" finished-at="2021-04-29T17:40:00Z" duration-ms="1151" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforemethod -->
        <test-method signature="DropDownTest()[pri:0, instance:com.automation.test.DropDown@5c90e579]" started-at="2021-04-29T17:40:00Z" name="DropDownTest" finished-at="2021-04-29T17:40:00Z" duration-ms="2" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.automation.test.DropDown.DropDownTest(DropDown.java:15)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DropDownTest -->
        <test-method is-config="true" signature="logout(org.testng.ITestResult)[pri:0, instance:com.automation.test.DropDown@5c90e579]" started-at="2021-04-29T17:40:00Z" name="logout" finished-at="2021-04-29T17:40:01Z" duration-ms="691" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=DropDownTest status=FAILURE method=DropDown.DropDownTest()[pri:0, instance:com.automation.test.DropDown@5c90e579] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
      </class> <!-- com.automation.test.DropDown -->
      <class name="com.automation.test.CheckBox">
        <test-method is-config="true" signature="beforemethod()[pri:0, instance:com.automation.test.CheckBox@4e3958e7]" started-at="2021-04-29T17:39:53Z" name="beforemethod" finished-at="2021-04-29T17:39:54Z" duration-ms="1112" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforemethod -->
        <test-method signature="Checkboxes()[pri:0, instance:com.automation.test.CheckBox@4e3958e7]" started-at="2021-04-29T17:39:54Z" name="Checkboxes" finished-at="2021-04-29T17:39:54Z" duration-ms="1" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.automation.test.CheckBox.Checkboxes(CheckBox.java:22)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Checkboxes -->
        <test-method is-config="true" signature="logout(org.testng.ITestResult)[pri:0, instance:com.automation.test.CheckBox@4e3958e7]" started-at="2021-04-29T17:39:54Z" name="logout" finished-at="2021-04-29T17:39:55Z" duration-ms="746" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Checkboxes status=FAILURE method=CheckBox.Checkboxes()[pri:0, instance:com.automation.test.CheckBox@4e3958e7] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
      </class> <!-- com.automation.test.CheckBox -->
      <class name="com.automation.test.DynamicContent">
        <test-method is-config="true" signature="beforemethod()[pri:0, instance:com.automation.test.DynamicContent@58ea606c]" started-at="2021-04-29T17:40:01Z" name="beforemethod" finished-at="2021-04-29T17:40:02Z" duration-ms="1181" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforemethod -->
        <test-method signature="DynamicContentTest()[pri:0, instance:com.automation.test.DynamicContent@58ea606c]" started-at="2021-04-29T17:40:02Z" name="DynamicContentTest" finished-at="2021-04-29T17:40:02Z" duration-ms="2" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.automation.test.DynamicContent.DynamicContentTest(DynamicContent.java:15)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DynamicContentTest -->
        <test-method is-config="true" signature="logout(org.testng.ITestResult)[pri:0, instance:com.automation.test.DynamicContent@58ea606c]" started-at="2021-04-29T17:40:02Z" name="logout" finished-at="2021-04-29T17:40:02Z" duration-ms="664" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=DynamicContentTest status=FAILURE method=DynamicContent.DynamicContentTest()[pri:0, instance:com.automation.test.DynamicContent@58ea606c] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
      </class> <!-- com.automation.test.DynamicContent -->
    </test> <!-- TestNgSmoke -->
  </suite> <!-- Suite1 -->
</testng-results>
